packages/frontend/mobile/src/components/sign-in/background.tsx

```diff
@@ -0,0 +1,203 @@
+import { useTheme } from 'next-themes';
+
+import * as styles from './background.css';
+
+const artsLight = `<svg viewBox="0 0 393 268" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
put it into a separate file and name it with `arts-light.inline.svg`, then add a rule in our [webpack config](./tools/cli/src/webpack/config.ts):

```ts
{
    test: /\.inline\.svg$/,
    type: 'asset/inline',
 },
```

And then u can import the file and use it in css.